OracleDatabase/RAC/OracleConnectionManager/README.md

```diff
@@ -1,30 +1,31 @@
-# Oracle Connection Manager on Docker
-Sample Docker build files to facilitate installation, configuration, and environment setup for DevOps users. For more information about Oracle Database please see the [Oracle Connection Manager online Documentation](http://docs.oracle.com/en/database/).
+# Oracle Connection Manager on Container
```

Comment:
Please change "on Container" to "Container Images"

Reply:
Changed and merged the file.